sample-docker-app/Dockerfile.arm64v8

```diff
@@ -0,0 +1,12 @@
+FROM node:12.19.0
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install -yq \
+  build-essential && \
+  apt-get clean && rm -rf /var/lib/apt/lists/*
+
+COPY package*.json ./
```

Comment:
I like the .gitignore idea :+1: 
```suggestion
```

Reply:
Right, now that we ignore the `node_modules`, we could `COPY . .` and `RUN npm install`.